index.js

```diff
@@ -135,6 +171,12 @@ function nowServing(line) {
   return `Currently serving ${line.shift()}.`
 }
 
+function takeATicketNumber(line) {
+    let ticket = line.length
+    console.log(`"Welcome. You are ticket number ${ticket}"`)
+    ticket++
+}
+
```

Comment:
I like the way you're thinking! However in this case, this will only work in some cases. By using `line.length` to determine the ticket number, you will be re-using ticket numbers